app/main.py

```diff
@@ -1,8 +1,48 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list[Car]) -> float:
+        income = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                income += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return round(income, 1)
+
+    def calculate_washing_price(self, car: Car) -> float:
+        washing_price = (
+            car.comfort_class * (
+                self.clean_power - car.clean_mark
+            ) * self.average_rating
+            / self.distance_from_city_center
+        )
+        return round(washing_price, 1)
+
+    def wash_single_car(self, car: Car) -> None:
+        car.clean_mark = self.clean_power
```

Comment:
![image](https://github.com/mate-academy/py-car-wash-station/assets/93856423/78552ff3-c40c-4eac-a84e-fb751156a90c)

Reply:
Victoria suggested to remove here. So I am a little bit confused 😅 
https://github.com/mate-academy/py-car-wash-station/pull/1064#discussion_r1485096788

Reply:
this is already checked in `serve_cars` before `wash_single_car` is called